src/components/layout.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import {graphql, StaticQuery} from 'gatsby';
+import {Layout} from '@dsmjs/components';
+import {node} from 'prop-types';
+import {css} from 'glamor';
+
+import 'prismjs/themes/prism-solarizedlight.css';
+
+export default function SiteLayout({children}) {
+  return (
+    <StaticQuery
+      query={graphql`
+      query LayoutQuery {
+        site {
+          siteMetadata {
+            title
+          }
+        }
+        allMarkdownRemark(
```

Comment:
thank you! figuring out this query has been on my list since first getting things kinda wired up